stdlib/public/core/VarArgs.swift

```diff
@@ -290,7 +290,8 @@ final public class VaListBuilder {
     // supported vararg type is greater than the alignment of Int.
     // FIXME: this implementation is not portable because
     // alignof differs from the ABI alignment on some architectures
-#if os(watchOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
+    // All ARM platforms with the exception of iOS use the AAPCS conventions
+#if !os(iOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
```

Comment:
Yeah, although I'd probably reverse the condition and maybe rephrase the whole comment to explain things better. That shouldn't block the merge, though.